tests/performance/test_cluster/test_cluster_performance/data/25w_50000a_thresholds.yaml

```diff
@@ -53,7 +53,7 @@ resources:
         workers:
           max: 47.85
           mean: 9.35
-          reg_cof: 0.06
+          reg_cof: 0.1
```

Comment:
Why is this change needed in this PR?

Reply:
The current thresholds are too small and strict, which causes most tests to be considered failed. It will be necessary to increase them little by little so that they only fail if an environment really has unusually high resource consumption.